javascript/classes.json

```diff
@@ -489,10 +489,22 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "81",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "javascript.options.experimental.private_fields",
+                  "value_to_set": "true"
+                },
+                {
+                  "type": "preference",
+                  "name": "javascript.options.experimental.private_methods",
+                  "value_to_set": "true"
+                }
+              ]
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "81"
```

Comment:
@chrisdavidmills I'm guessing this is also behind a preference on Android?

Reply:
Yup, added!